src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -150,8 +151,8 @@ public boolean link() throws IOException, InterruptedException {
         Path dalvikLibPath = dalvikPath.resolve("lib");
         Path dalvikLibArm64Path = dalvikLibPath.resolve("arm64-v8a");
 
-        String unalignedApk = dalvikBinPath.resolve("hello.unanligned.apk").toString();
-        String alignedApk = dalvikBinPath.resolve("hello.apk").toString();
+        String unalignedApk = dalvikBinPath.resolve(projectConfiguration.getAppName()+".unanligned.apk").toString();
```

Comment:
Might be a good time to fix the typo in the filename : `unanligned` --> `unaligned`.